docs/translations/api-docs/data-grid/data-grid/data-grid.json

```diff
@@ -816,6 +816,11 @@
       "nodeName": "the main container element",
       "conditions": "it has right pinned columns"
     },
+    "main--hasSkeletonLoadingOverlay": {
+      "description": "Styles applied to {{nodeName}} when {{conditions}}.",
+      "nodeName": "the main container element",
+      "conditions": "it has an active skeleton loading overlay"
+    },
```

Comment:
Can we remove these from the public API? These don't need to be there. I think you can add the flag `@ignore - Do not document` in the jsdoc for that.

Reply:
Yeah these don't need to be there, thanks. Fixed here https://github.com/mui/mui-x/pull/13293/commits/7c226b688013eba8dcc46a8617405c00518956f1